build_node/models.py

```diff
@@ -41,9 +41,10 @@ class Task(BaseModel):
     platform: TaskPlatform
     created_by: TaskCreatedBy
     repositories: typing.List[TaskRepo]
+    builted_srpm_url: typing.Optional[str]
 
     def is_srpm_build_required(self):
-        return not self.ref.url.endswith('src.rpm')
+        return not self.ref.url.endswith('src.rpm') and not self.builted_srpm_url
```

Comment:
JFYI. NOT a AND NOT b could be changed to NOT (a OR b)

Reply:
Thanks, i'll wix it

Reply:
Fixed